.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+### Description
+
+<!-- Describe your changes in detail -->
+
+### Motivation & Context
+
+<!-- Why is this change required? What problem does it solve? -->
+
+### Types of changes
```

Comment:
There is maybe an unwanted side effect with checkboxes here.
Considering the PR filled this way:

![Screenshot from 2021-12-30 07-37-28](https://user-images.githubusercontent.com/17381666/147727779-3a7fcd65-1a14-48fd-8550-4bfc9174df89.png)

It will be considered as a checklist by GitHub and will be displayed like that in the PRs list:

![Screenshot from 2021-12-30 07-38-53](https://user-images.githubusercontent.com/17381666/147727840-26cda4f8-08f4-4329-9272-c0327aad412e.png)

"7 of 10 tasks" may be troubling or frustrating and can't be fixed by the user since they are not real tasks but choices.

If you find this comment relevant, we could try another approach like that:

```markdown
<!-- What types of changes does your code introduce? Uncomment the following types that apply. -->

<!-- - Bug fix (non-breaking change which fixes an issue) -->
<!-- - New feature (non-breaking change which adds functionality) -->
<!-- - Refactoring (non-breaking change) -->
<!-- - Breaking change (fix or feature that would change existing functionality) -->
```